operator/internal/handlers/lokistack_create_or_update.go

```diff
@@ -77,6 +79,13 @@ func CreateOrUpdateLokiStack(
 		}
 	}
 
+	schemas, err := storageschema.GetLokiStorageSchemaData(ctx, k, req)
+	if err != nil {
+		log.Error(err, "Failed to retrieve config map for storage schemas.")
+		return err
+	}
+	objStore.Schemas = storage.UpdateSchemas(stack.Spec.Storage.SchemaVersion, schemas)
```

Comment:
Does the current code handle the situation when the `SchemaVersion` is not set in the `LokiStack`? I think, if we want to keep existing stacks then this would need to be handled like it was set to `v11`.

Reply:
The default covers the case where the user does not specifically set the version.